dev/MRTCore/mrt/Core/unittests/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Taef" version="10.75.221207001" targetFramework="native" />
+  <package id="Microsoft.Taef" version="10.58.210222006-develop" targetFramework="native" />
```

Comment:
I see in the PR description that reverting the version of TAEF is intended. But this is a fairly big jump backwards, 20+ months, is this move explained somewhere?
Also I wasn't able to find "10.58.210222006-develop" here: https://www.osgwiki.com/wiki/TAEF_Release_Notes#Fix_Use-after-free_when_Appending_a_String_to_Itself, so I'm unclear why it is picked as opposed to, say, v10.58.210305002 mentioned there.

Reply:
10.58.210222006-develop is no more :)